projects/demo/src/modules/components/tag/examples/4/index.html

```diff
@@ -0,0 +1,12 @@
+<ng-container *ngFor="let tag of tags; let index = index">
+    <tui-tag
+        size="l"
+        status="custom"
+        class="tag"
+        [value]="tag"
+        [hoverable]="true"
+        [editable]="true"
+        [removable]="tags.length > 1"
```

Comment:
Not sure this makes sense, since if it's always editable — you can always remove it by editing it to an empty line.

Reply:
Let's change it to uneditable when it comes to 1 item.

Reply:
Why not just keep them always removable? That would be weird that you could edit an items while it's not the last one and then if you remove all others it becomes uneditable.

Reply:
Just in this example

Reply:
`That would be weird that you could edit an items while it's not the last one` I agree with that